### DISCUSSION TITLE
Chess - Display of the boear and more

### DISCUSSION THREAD

**Discussion by DavidDHBW** [DavidDHBW/codespaces-jupyter#13] [General]
This week,after i fixed the ALSA lib error, i added many new functionality to my chess game.
1)the Board is now drawed correctly
2)you can make a move by clicking on a piece and the destination square
3)the move is stored in a move log file, that way you can later undo changes or analyse your game after.

Now i have to implement the logic, since currently you could make any move and cant castle.
[code](https://github.com/DavidDHBW/codespaces-jupyter/tree/main/Python/Chess)

**Comment by LxisW** [comment]
Very nice idea, looking forward to see more about chess since I am not that familiar with it!

**Comment by emmanuelyoussefyoussef** [comment]
Great idea! Now, VS Code isn't as dull as it used to be.
